src/phpdoc/parser.go

```diff
@@ -9,8 +9,9 @@ import (
 )
 
 type Comment struct {
-	Raw    string
-	Parsed []CommentPart
+	Raw     string
+	Parsed  []CommentPart
+	Inherit bool
```

Comment:
Maybe add an in-line comment that it means that there was `inheritdoc` somewhere in the parsed comment?
```
Inherit bool // comment contains @inheritdoc marker
```

Reply:
Done.